### DISCUSSION TITLE
Overhead Door

### DISCUSSION THREAD

**Discussion by ColtzNut** [home-assistant/frontend#16517] [Automations / scripts / scenes] [answered]
We recently moved into a new home and the garage doors are controlled by an app and I can open, close and see door status from this app.  I know it works with Google, but I don't see anything for HA.  The iOS app is called OHD Anywhere.  Is there a place to request an integration?  Or, better yet, I'm hoping I just missed it.

**Comment by karwosts** [comment, marked as answer]
Seems to be something suggested here: https://community.home-assistant.io/t/ohd-anywhere-app/313127

Feature requests like this one not related to frontend should go in the feature requests topic of the forum.
